apps/vela-mobile/scripts/sync-ios-version.mjs

```
@@ -0,0 +1,127 @@
+
+const root = resolve(dirname(fileURLToPath(import.meta.url)), '..');
+const pkgPath = resolve(root, 'package.json');
+const pbxprojPath = resolve(
+  root,
+  'src-capacitor/ios/App/App.xcodeproj/project.pbxproj',
+);
+
+// Vite's `mode` determines which mode-specific env files are loaded
+// (`development` for `quasar dev`, `production` for `quasar build`). Default to
+// `production` since iOS MARKETING_VERSION matters most for release builds.
+const argvMode = parseArg(process.argv.slice(2), 'mode');
```

ESLint

[error] 'process' is not defined.

(no-undef)